src/test/runner/validation/and.test.ts

```diff
@@ -0,0 +1,170 @@
+import { mock, MockProxy } from "jest-mock-extended";
+
+import { PullRequestApi } from "../../../github/pullRequest";
+import { TeamApi } from "../../../github/teams";
+import { ConfigurationFile, RuleTypes } from "../../../rules/types";
+import { ActionRunner } from "../../../runner";
+import { TestLogger } from "../../logger";
+
+describe("'And' rule validation", () => {
+  let api: MockProxy<PullRequestApi>;
+  let teamsApi: MockProxy<TeamApi>;
+  let runner: ActionRunner;
+  let logger: TestLogger;
+  const users = ["user-1", "user-2", "user-3"];
+  beforeEach(() => {
+    logger = new TestLogger();
+    api = mock<PullRequestApi>();
+    teamsApi = mock<TeamApi>();
+    teamsApi.getTeamMembers.calledWith("abc").mockResolvedValue(users);
+    api.listModifiedFiles.mockResolvedValue([".github/workflows/review-bot.yml"]);
+    api.listApprovedReviewsAuthors.mockResolvedValue([]);
+    runner = new ActionRunner(api, teamsApi, logger);
+  });
+
+  describe("approvals", () => {
+    test("should return true if the reviewers reviewed", async () => {
+      const config: ConfigurationFile = {
+        rules: [
+          {
+            name: "And rule",
+            type: RuleTypes.And,
+            condition: { include: ["review-bot.yml"] },
+            reviewers: [{ teams: ["abc"], min_approvals: 1 }],
+          },
+        ],
+      };
+      api.listApprovedReviewsAuthors.mockResolvedValue([users[0]]);
+      const evaluation = await runner.validatePullRequest(config);
+      expect(evaluation).toHaveLength(0);
```

Comment:
> should return true if the reviewers reviewed

> `      expect(evaluation).toHaveLength(0);`

Feels like the `return true` part is outdated

Reply:
![image](https://media3.giphy.com/media/JUepN05v2gDntNAaLI/giphy.gif?cid=ecf05e474nk30g8j35jwfao1zb16bn4keznf4qc9szcnhhbw&ep=v1_gifs_search&rid=giphy.gif&ct=g)